core-services/prow/02_config/openshift/cluster-openshift-apiserver-operator/_pluginconfig.yaml

```diff
@@ -6,6 +6,13 @@ label:
       - wangke19
       - gangwgr
       label: cherry-pick-approved
+    - allowed_users:
+      - adambkaplan
```

Comment:
I shouldn't be on this list, as the Build + Jenkins team doesn't own the openshift-apiserver-operator